testing/benchmark/system-profiles/16GBx2zone.tfvars

```diff
@@ -3,10 +3,12 @@ user_name = "USER"
 # The number of AZs the APM Server should span.
 apm_server_zone_count = 1
 # The Elasticsearch cluster node size.
-elasticsearch_size = "64g"
+elasticsearch_size = "128g"
```

Comment:
Have you also tried with an even larger ES instance size + doubling the apm-shards? 
`256GB`, `8` shards?

Reply:
I have, for 16GB, and there's no difference.